frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -317,7 +318,7 @@ export default ({
       this.$router.push({
         name: 'GroupChatConversation',
         params: { chatRoomID: obj.chatroomId }
-      })
+      }).catch(logExceptNavigationDuplicated)
```

Comment:
While this might hide the error, I don't think it will fix the issue, which is that we're supposed to switch to the #general chatroom and it doesn't

Reply:
Oh, do you mean it doesn't redirect to the `#general` chatroom? On my side, it redirects well. I will check several times more again.

Reply:
> A NavigationDuplicated error appears, even though we are not in the general chatroom.

I can see the `#general` chatroom link has the link to the `#new-chatroom`. Two chatrooms IDs are same; one is from the error message and the other is from the current tab link.

But I can't reproduce this error actually.

Reply:
Huh. OK I can't reproduce it now anymore either. Weird. 🤷‍♂️

Reply:
@taoeffect, I think this error is already fixed with [this commit](https://github.com/okTurtles/group-income/pull/1965/commits/4c5ba994c1d83abff88d42563d54c71de7f63455). Can you please test it again and keep me updated?

Reply:
I mentioned above that I can't reproduce anymore